### PR TITLE
cmeps integration changes

### PR DESCRIPTION
**Pull request by DeniseWorthen**
* issue #48 : Changes two field names for the nems mediator for consistency w/ cmeps (mean_zonal_moment_flx_atm and mean_merid_moment_flux). An additional 3 field names are added to the NUOPC dictionary for the NEMS mediator but these are not currently being used.

Also updates some CMEPS configuration consistent with current master of CMEPS. 

Related PRs:
fv3atm PR [#104](https://github.com/NOAA-EMC/fv3atm/pull/104)
CCPP physics PR [#440](https://github.com/NCAR/ccpp-physics/pull/440) 
ufs-weather-model PR [#108](https://github.com/ufs-community/ufs-weather-model/pull/108)

### CODE REVIEW

File: src/module_EARTH_GRID_COMP.F90 [hunk 1]
```diff
@@ -334,6 +356,17 @@ SUBROUTINE EARTH_REGISTER(EARTH_GRID_COMP,RC_REG)
           file=__FILE__)) &
           return  ! bail out
       endif
+      if (.not.NUOPC_FieldDictionaryHasEntry( &
+        "mean_sensi_heat_flx_atm")) then
```
Comment (junwang-noaa) [inline src/module_EARTH_GRID_COMP.F90:360]: Denise, we do not have this change on "mean_sensi_heat_flx_atm" and "mean_laten_heat_flx_atm" in fv3atm, do we need  to change them in fv3 too?

**Comment by DeniseWorthen** [comment]
We do not need them in FV3. The name used by FV3 (mean_sensi_heat_flx) is still defined in the Dictionary and is used to define the fldsFrAtm in the mediator. Only the momentum fluxes need to be changed in the mediator and FV3 to be consistent w/ the CMEPS exchange fields.

**Comment by junwang-noaa** [comment]
> * issue #48 : Changes two field names for the nems mediator for consistency w/ cmeps (mean_zonal_moment_flx_atm and mean_merid_moment_flux). An additional 3 field names are added to the NUOPC dictionary for the NEMS mediator but these are not currently being used.
> 
> Also updates some CMEPS configuration consistent with current master of CMEPS.

Denise,
 Please add fv3atnm PR #104, CCPP physics PR #440, ufs-weather-model PR #108 to this ticket. Thanks.

**Review by junwang-noaa — APPROVED** [review]

**Comment by junwang-noaa** [comment]
Final RT passed, code is ready for commit after review. Thanks

**Pull request merged**